src/fatal.cc

```diff
@@ -16,19 +16,8 @@
 static void
 fatal_common(const char *message)
 {
-#if HAVE_SYSLOG
-    syslog(LOG_ALERT, "%s", message);
-#endif
-
-    fprintf(debug_log, "FATAL: %s\n", message);
-
-    if (Debug::log_stderr > 0 && debug_log != stderr)
-        fprintf(stderr, "FATAL: %s\n", message);
-
-    fprintf(debug_log, "Squid Cache (Version %s): Terminated abnormally.\n",
-            version_string);
-
-    fflush(debug_log);
```

Comment:
What is our justification for removing fflush()? Is that because debug_log is always line-buffered? Can you tell why that line was originally added here?

Reply:
The debugs will call the fflush so it is not needed here.
Always after writing to debug_log we are calling fflush. I suppose this is done to avoid lost messages after an unexpected squid exit.

Reply:
Ah, even better!

Reply:
Alex, all of your requests for changes should addressed in my latest patch. 
Please  review.